app/controllers/graphql_controller.rb

```diff
@@ -35,7 +35,7 @@ def execute
   private
 
   def graphql_params
-    params.permit(:query, :variables, :operationName)
+    params.permit(:query, :operationName)
```

Comment:
@hpjaj since we're doing `params.dig` to get our variables, I dont think we need this here anymore

Reply:
👍